installation/README.md

```diff
@@ -0,0 +1,93 @@
+# Installing Config Sync manually
+
+This document is a guide on how to manually install Config Sync on a Kubernetes
+cluster without Fleet Management.
+
+The recommended approach is to [install Config Sync managed by fleet](https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/installing-config-sync).
```

Comment:
Might be fine to remove this paragraph given that this doc lives in OSS. We will need to update the GCP doc here anyways https://cloud.google.com/kubernetes-engine/enterprise/config-sync/docs/how-to/installing-kubectl

Note that we will not redirect GCP users to read manual installation guide from OSS, nor downloading from the OSS GitHub page.

Reply:
The primary intent of this README is to be bundled in a tarball alongside the `kustomization.yaml` and uninstall script as a release artifact, so it won't necessarily be exclusive to the github repo. If you prefer this can be excluded from the release artifact and the instructions can be hosted on the gcloud docs.

> Note that we will not redirect GCP users to read manual installation guide from OSS, nor downloading from the OSS GitHub page.

How do you mean for the release artifact to be downloaded? GCS? The intent is to publish this release artifact to both the github release page and to GCS.

Reply:
My comment is mainly about: 1) similar content will be needed in the GCP doc, and 2) we will not ask GCP customers to come to GitHub to figure out how to do manual installation themselves

Reply:
Removed in https://github.com/GoogleContainerTools/kpt-config-sync/pull/1456